### PR TITLE
repair citation on foreset_fires_portugal

### PR DESCRIPTION
**Pull request by henrykironde**
The citation string had some byte representation for one of the non ascii characters.

### CODE REVIEW

File: scripts/forest_fires_portugal.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "citation": "[Cortez and Morais, 2007] P. Cortez and A. Morais. A Data Mining Approach to Predict Forest Fires using Meteorological Data. In J. Neves, M. F. Santos and J. Machado Eds., New Trends in Artificial Intelligence, Proceedings of the 13th EPIA 2007 - Portuguese Conference on Artificial Intelligence, December, Guimar\u00e3es, Portugal, pp. 512-523, 2007. APPIA, ISBN-13 978-989-95618-0-9.",
+    "citation": "Cortez and Morais, 2007] P. Cortez and A. Morais. A Data Mining Approach to Predict Forest Fires using Meteorological Data. In J. Neves, M. F. Santos and J. Machado Eds., New Trends in Artificial Intelligence, Proceedings of the 13th EPIA 2007 - Portuguese Conference on Artificial Intelligence, December, Guimarães, Portugal, pp. 512-523, 2007. APPIA, ISBN-13 978-989-95618-0-9.",
```
Comment (ethanwhite) [inline scripts/forest_fires_portugal.json]: How about we just make the beginning "Cortez, P. and A. Morais. 2007. A Data Mining..."

File: scripts/forest_fires_portugal.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "citation": "[Cortez and Morais, 2007] P. Cortez and A. Morais. A Data Mining Approach to Predict Forest Fires using Meteorological Data. In J. Neves, M. F. Santos and J. Machado Eds., New Trends in Artificial Intelligence, Proceedings of the 13th EPIA 2007 - Portuguese Conference on Artificial Intelligence, December, Guimar\u00e3es, Portugal, pp. 512-523, 2007. APPIA, ISBN-13 978-989-95618-0-9.",
+    "citation": "Cortez and A. Morais. A Data Mining Approach to Predict Forest Fires using Meteorological Data. In J. Neves, M. F. Santos and J. Machado Eds., New Trends in Artificial Intelligence, Proceedings of the 13th EPIA 2007 - Portuguese Conference on Artificial Intelligence, December, Guimarães, Portugal, pp. 512-523, 2007. APPIA, ISBN-13 978-989-95618-0-9.",
```
Comment (henrykironde) [inline scripts/forest_fires_portugal.json]: not yet clean

File: scripts/forest_fires_portugal.json (hunk 1 above)
Comment (henrykironde) [inline scripts/forest_fires_portugal.json]: made it to P. Cortez and A. Morais. A D ... since this is how it appears

**Comment by ethanwhite** [comment]
:+1:

**Pull request merged**